spec/Section 3 -- Type System.md

```diff
@@ -232,6 +232,152 @@ Schema extensions have the potential to be invalid if incorrectly defined.
 2. Any non-repeatable directives provided must not already apply to the
    original Schema.
 
+### Schema Coordinates
+
+Schema Coordinates are human readable strings that uniquely identify an element defined in a GraphQL Schema.
+
+**Definition**
+
+SchemaCoordinate:
+  - Name
+  - Name . Name
+  - Name . Name ( Name : )
+  - @ Name
+  - @ Name ( Name : )
+
+**Semantics**
+
+SchemaCoordinate: Name
+  1. Let {typeName} be the value of the first {Name}.
+  2. Return the type in the schema named {typeName}.
+
+SchemaCoordinate: Name . Name
+  1. Let {typeName} be the value of the first {Name}.
+  2. Let {type} be the type in the schema named {typeName}.
+  3. If {type} is an Enum type:
+    1. Let {enumName} be the value of the second {Name}.
+    2. Return the enum value of {type} named {enumName}.
```

Comment:
I wonder if this should be `enumValueName` rather than `enumName` (which sounds to me like the type name).